absl/debugging/symbolize_elf.inc

```diff
@@ -1451,7 +1454,7 @@ static bool MaybeInitializeObjFile(ObjFile *obj) {
       }
       phoff += phentsize;
 
-#if defined(__powerpc__) && !(_CALL_ELF > 1)
+#if HAVE_PPC64_ELFV2_ABI
```

Comment:
How was this change tested? I don't have access to any PPC machine so I can't test it myself.

The branch looks like it was previously for the V1 ABI (the condition was `!(_CALL_ELF > 1)`), but now appears to be the V2 ABI.

Reply:
I have an Apple G5 Mac that runs Debian Linux.

The comments surrounding this change don't match the behaviour -- the previous check was over-broad.
32-bit powerpc never implemented ELF ABIv2.

Reply:
That said, I am now wondering if you are correct and I have inverted the check.

Reply:
> That said, I am now wondering if you are correct and I have accidentally inverted the check. I note that 32-bit binaries built with gcc 14.2.0 on my machine do not appear to generate `opd` ELF sections.

Aaahhh, a quick search online shows that `opd` is only generated for 64-bit object files in ABIv1.